src/Simplify.elm

```diff
@@ -1372,6 +1375,9 @@ expressionVisitorHelp node context =
                 Expression.RecordUpdateExpression (Node recordNameRange _) setters ->
                     onlyErrors (recordAccessChecks (Node.range node) (Just recordNameRange) fieldName setters)
 
+                Expression.ParenthesizedExpression (Node _ (Expression.LetExpression { expression })) ->
```

Comment:
Instead of doing parens here, you can also do `case Node.value (AstHelpers.removeParens record) of` a few lines above. I do think we should then add a few tests for the other cases to see whether they support parens ok.

Using this function can be nicer because we can then pile on more simplifications in `--fix-all` before `elm-format` kicks in (and asks one less prompt to the user in this situation).

But if this makes the fix a lot more annoying, let me know and let's skip it. Do you mind trying this out?